docs/content/docs/custom-resource/pod-template.md

```diff
@@ -93,16 +90,18 @@ spec:
 ```
 
 {{< hint info >}}
-When using the operator with Flink native Kubernetes integration, please refer to [pod template field precedence](
-https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/#fields-overwritten-by-flink).
+当使用具有 Flink 原生 Kubernetes 集成的 operator 时，请参考 [pod template 字段优先级](
+https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/#fields-overwritten-by-flink)。
 {{< /hint >}}
 
+
 ## Array Merging Behaviour
 
-When layering pod templates (defining both a top level and jobmanager specific podtemplate for example) the corresponding yamls are merged together.
+<a name="array-meging-behaviour"></a>
+
+当分层 pod templates（例如同时定义顶级和任务管理器特定的 pod 模板）时，相应的 yaml 会合并在一起。
 
-The default behaviour of the pod template mechanism is to merge array arrays by merging the objects in the respective array positions.
-This requires that containers in the podTemplates are defined in the same order otherwise the results may be undefined.
+Pod 模板机制的默认行为是通过合并相应数组位置的对象合并 json 类型的数组。
```

Comment:
```suggestion
Pod 模板机制的默认行为是通过合并相应数组位置的对象来合并数组。
```

Why introducing json?

Reply:
Because I'm sure array arrays translates to

Reply:
I tried to convert it using json formatter just now, it's not json.

![image](https://github.com/apache/flink-kubernetes-operator/assets/38427477/f44ad700-4348-4e77-962c-efebfec04b61)